Vagrantfile

```diff
@@ -1,35 +1,45 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+require 'fileutils'
+
 netplugin_synced_gopath="/opt/golang"
-host_gobin_path="/opt/go/bin"
-host_goroot_path="/opt/go/root"
+FileUtils.cp "/etc/resolv.conf", Dir.pwd
 
 provision_common = <<SCRIPT
 ## setup the environment file. Export the env-vars passed as args to 'vagrant up'
 echo Args passed: [[ $@ ]]
+
+echo -n "$1" > /etc/hostname
+hostname -F /etc/hostname
+
+/sbin/ip addr add "$3/24" dev eth1
+/sbin/ip link set eth1 up
+/sbin/ip link set eth2 up
+
 echo 'export GOPATH=#{netplugin_synced_gopath}' > /etc/profile.d/envvar.sh
 echo 'export GOBIN=$GOPATH/bin' >> /etc/profile.d/envvar.sh
 echo 'export GOSRC=$GOPATH/src' >> /etc/profile.d/envvar.sh
-echo 'export GOROOT=#{host_goroot_path}' >> /etc/profile.d/envvar.sh
-echo 'export PATH=$PATH:#{host_goroot_path}/bin:#{host_gobin_path}:$GOBIN' >> /etc/profile.d/envvar.sh
-if [ $# -gt 0 ]; then
-    echo "export $@" >> /etc/profile.d/envvar.sh
```

Comment:
I think it's still in there... https://github.com/contiv/netplugin/pull/133/files#diff-23b6f443c01ea2efcb4f36eedfea9089R40